setup.cfg

```diff
@@ -59,6 +59,8 @@ dev =
     pytest-cov
     mock
     ipykernel
+    sphinx == 6.1.3
+    sphinx-autodoc-typehints == 1.22.0
```

Comment:
I read somewhere that you should pin exact docs versions to ensure consistent builds moving forward, which I think makes sense